lib/rules/selector-not-notation/index.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const isStandardSyntaxRule = require('../../utils/isStandardSyntaxRule');
+const parseSelector = require('../../utils/parseSelector');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+const {
+	isPseudoClass: buggyIsPseudoClass, // cf https://github.com/postcss/postcss-selector-parser/blob/master/src/selectors/guards.js#L52
```

Comment:
will have to be fixed upstream
no rule except this one uses this util, so we are safe
the extension at line 30 is future proof

Reply:
I've opened PR https://github.com/postcss/postcss-selector-parser/pull/256 to fix the bug.

Reply:
[suggest] What if making the comment TODO like this?

```suggestion
	isPseudoClass: buggyIsPseudoClass, // TODO: https://github.com/postcss/postcss-selector-parser/pull/256
```